internetarchive/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '3.0.2'
+__version__ = '3.0.3'
```

Comment:
You should not change the version. That's what Jake does when preparing a new release.

Reply:
@maxz Sorry about that. This change has been reverted. Thanks!